OpenRA.Game/Traits/World/DebugVisualizations.cs

```diff
@@ -20,7 +20,49 @@ public class DebugVisualizations
 		public bool CombatGeometry;
 		public bool RenderGeometry;
 		public bool ScreenMap;
-		public bool DepthBuffer;
 		public bool ActorTags;
+
+		bool depthBufferDirty = true; // Force normal rendering at the start of a new world
```

Comment:
```suggestion
		// Force normal rendering at the start of a new world
		bool depthBufferDirty = true;
```

Reply:
Done, and also improved the wording of the comment.